### PR TITLE
[WIP] redesign the whole frontend application

### PR DESCRIPTION
**Pull request by Rintel**
New frontend design! This branch basically touches everything.

This is (as labeled) pretty much work in progress.  We have a lot of dependencies because of these changes. Don't merge!

### CODE REVIEW

File: web/src/components/common/SideBar.jsx [hunk 1]
```diff
@@ -17,8 +17,9 @@ export default class SideBar extends Component {
     };
   }
 
+  //has to be updated if history is working
   render() {
-    let isActive = (url) => { url === this.props.location };
+    let isActive = (url) => false;//{ url === location };
```
Comment (Drakulix) [inline web/src/components/common/SideBar.jsx]: why?

Comment (mharasic) [reply web/src/components/common/SideBar.jsx]: Because of an awesome bug in locations... now returned

**Comment by mharasic** [comment]
Still work in progress as it has to be connected to the backend

**Review by Drakulix — APPROVED** [review]

**Comment by Drakulix** [comment]
:hankey:

**Review by Drakulix — DISMISSED** [review]
:hankey: :gun:

**Pull request merged**